land_claims.civmap.json

```diff
@@ -241,5 +241,13 @@
+{"id":"ac713034-c3b1-4654-bd22-c9370ec06fa6","collectionId":"civmap:collection/user","polygon":[[[7586,-7109],[7925,-7106],[7931,-6861],[7638,-6887],[7548,-7025],[7581,-7106]]],"name":"Union (Buckeye)"},
+{"id":"b56e0ff0-3ad0-4df8-ab44-f0e769d38acf","collectionId":"civmap:collection/user","polygon":[[[3119,12727],[3054,12683],[3062,12599],[3061,12569],[3136,12494],[3202,12460],[3215,12368],[3267,12343],[3462,12354],[3678,12433],[3727,12512],[3371,12760]]],"name":"Fight Club Island"},
+{"id":"082fa540-1763-4623-ae00-73f2ad277ac9","collectionId":"civmap:collection/user","polygon":[[[4864,12523],[-4288,12483],[-11008,7203],[-12352,3614],[-13120,89],[-11904,-3840],[-10496,-7089],[-7808,-10243],[-4928,-12280],[-1344,-13056],[1536,-12921],[5824,-11959],[8576,-9460],[10560,-7025],[12096,-3884],[13376,-552],[13376,1115],[12736,2397],[12160,4447],[11200,7331],[8576,10472],[4928,12288]]]},
+{"id":"2ea699c2-6a97-406c-93ac-d5cc73ca5801","collectionId":"civmap:collection/user","polygon":[[[5902,910],[5877,923],[5874,931],[5869,938],[5850,920],[5849,573],[5728,450],[5651,452],[5649,217],[5720,164],[5734,165],[5747,168],[5753,181],[5790,166],[5802,157],[5828,137],[5814,100],[5828,91],[5834,72],[5849,62],[5863,53],[5876,58],[5882,68],[5902,56],[5903,150],[5902,244]]],"name":"Partyland 2: The Sequel"},
```

Comment:
we should display a `"contact"` or `"creator"` on every entry; would be cool if you added them now, otherwise we can also do this when adding it to all other entries later. tracked in #173